src/bgpants.js

```diff
@@ -77,9 +77,7 @@
             var endBean = 4368;
             if(startBean < BEAN && BEAN < endBean)
             {
-              let beanInterval = endBean - startBean;
-              let howFarWeAreAlong = BEAN - startBean;
-              let fractionIn = howFarWeAreAlong / beanInterval;
+              var fractionIn = (BEAN - startBean) / (endBean - startBean);
```

Comment:
This will not give you a smooth 60 FPS animation. Bean is incremented less often than 60 fps.

A common approach is using `frame` with `FRAME_FOR_BEAN` instead

Reply:
Rewritten, that will be `(this.frame - FRAME_FOR_BEAN(startBean)) / (FRAME_FOR_BEAN(endBean) - FRAME_FOR_BEAN(startBean))`